content/reference/v1beta1/repeatable.md

```diff
@@ -0,0 +1,304 @@
+      items:
+      - name: deployment-name
+        title: Deployment Names
+        type: text
+        repeatable: true
+        template:
+        - apiVersion: apps/v1 #processed fourth
+          kind: Deployment
+          name: my-deployment
+          namespace: my-app
+          yamlPath:
+        {other item properties ...}
+```
+
+# Examples
+
+In this example, the default service port of "80" is included with the release. Port 443 is added as an additional port on the KOTSADM `Config` page, which is stored in the ConfigValues file.
+## Repeatable Item Example for a YamlPath
+**Config spec**
+```yaml
+    - name: ports
+      items:
+      - name: service_port
+        title: Service Port
+        type: text
+        repeatable: true
+        template:
+        - apiVersion: v1
+          kind: Service
+          name: my-service
+          namespace: my-app
+          yamlPath: spec.ports[0]
+        valuesByGroup:
+          ports:
+            port-default-1: "80"
+```
+
+**Config values**
+```yaml
+apiVersion: kots.io/v1beta1
+kind: ConfigValues
+metadata:
+  name: example_app
+spec:
+  values:
+    port-default-1:
+      repeatableItem: service_port
+      value: "80"
+    service_port-8jdn2bgd:
+      repeatableItem: service_port
+      value: "443"
+```
+
+**Template manifest**
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+  namespace: my-app
+spec:
+  type: NodePort
+  ports:
+  - port: repl{{ ConfigOption "[[repl .service_port ]]" | ParseInt }}
+    name: '[[repl .service_port ]]'
+  selector:
+    app: repeat_example
+    component: my-deployment
+```
+
+**After repeatable config processing**
+
+*Note: this phase is internal to KOTS configuration rendering. This example is only provided to further explain the templating process.*
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+  namespace: my-app
+spec:
+  type: NodePort
+  ports:
+  - port: repl{{ ConfigOption "port-default-1" | ParseInt }}
+    name: 'port-default-1'
+  - port: repl{{ ConfigOption "service_port-8jdn2bgd" | ParseInt }}
+    name: 'service_port-8jdn2bgd'
+  selector:
+    app: repeat_example
+    component: my-deployment
+```
+
+**Resulting manifest**
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+  namespace: my-app
+spec:
+  type: NodePort
+  ports:
+  - port: 80
+    name: port-default-1
+  - port: 443
+    name: service_port-8jdn2bgd
+  selector:
+    app: repeat_example
+    component: my-deployment
+```
+
+## Repeatable Item Example for an Entire Document
+**Config spec**
+```yaml
+    - name: ports
+      items:
+      - name: service_port
+        title: Service Port
+        type: text
+        repeatable: true
+        template:
+        - apiVersion: v1
+          kind: Service
+          name: my-service
+          namespace: my-app
+          yamlPath: ""
+        valuesByGroup:
+          ports:
+            port-default-1: "80"
+            service_port-8jdn2bgd: "443"
```

Comment:
Shouldn't this match the first example and not include `service_port-8jdn2bgd: "443"` here since the vendor wouldn't write that? And then you'd have a Config values section that merges the spec with the user input.

Reply:
yep, updated one example but not the other, thanks!